CP02PMCO/CP02PMCO_D00010_ingest.csv

```diff
@@ -1,10 +1,10 @@
-mi.dataset.driver.adcps_jln.stc.adcps_jln_stc_telemetered_driver,/omc_data/whoi/OMC/CP02PMCO/D00010/imm/adcpt/adcpt*.DAT,CP02PMCO-RII01-02-ADCPTG010,telemetered,Available,
-#mi.dataset.driver.cg_stc_eng.stc.cg_stc_eng_stc_telemetered_driver,/omc_data/whoi/OMC/CP02PMCO/D00010/irid2shore/stc_status.*_*.txt,CP02PMCO-SBS01-00-STCENG000,telemetered,Available,Not ingested due to system error
-#mi.dataset.driver.cg_stc_eng.stc.mopak_o_dcl_telemetered_driver,/omc_data/whoi/OMC/CP02PMCO/D00010/3dmgx3/*_*.3dmgx3.log,CP02PMCO-SBS01-01-MOPAK0000,telemetered,Available,Not ingested due to system error
-mi.dataset.driver.WFP_ENG.STC_IMODEM.wfp_eng_stc_imodem_telemetered_driver,/omc_data/whoi/OMC/CP02PMCO/D00010/imm/mmp/E*.DAT,CP02PMCO-WFP01-00-WFPENG000,telemetered,Available,
-mi.dataset.driver.vel3d_k.wfp_stc.vel3d_k_wfp_stc_telemetered_driver,/omc_data/whoi/OMC/CP02PMCO/D00010/imm/mmp/A*.DEC,CP02PMCO-WFP01-01-VEL3DK000,telemetered,Available,
-mi.dataset.driver.dofst_k.wfp.coastal_dofst_k_wfp_telemetered_driver,/omc_data/whoi/OMC/CP02PMCO/D00010/imm/mmp/C*.DAT,CP02PMCO-WFP01-02-DOFSTK000,telemetered,Available,
-mi.dataset.driver.ctdpf_ckl.wfp.coastal_ctdpf_ckl_wfp_telemetered_driver,/omc_data/whoi/OMC/CP02PMCO/D00010/imm/mmp/C*.DAT,CP02PMCO-WFP01-03-CTDPFK000,telemetered,Available,
-mi.dataset.driver.flort_kn.stc_imodem.flort_kn__stc_imodem_telemetered_driver,/omc_data/whoi/OMC/CP02PMCO/D00010/imm/mmp/E*.DAT,CP02PMCO-WFP01-04-FLORTK000,telemetered,Available,
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.adcps_jln.stc.adcps_jln_stc_telemetered_driver,/omc_data/whoi/OMC/CP02PMCO/D00010/imm/adcpt/adcpt*.DAT,CP02PMCO-RII01-02-ADCPTG010,telemetered,Available,
+mi.dataset.driver.cg_stc_eng.stc.cg_stc_eng_stc_telemetered_driver,/omc_data/whoi/OMC/CP02PMCO/D00010/irid2shore/stc_status.*_*.txt,CP02PMCO-SBS01-00-STCENG000,telemetered,Available,
+mi.dataset.driver.cg_stc_eng.stc.mopak_o_dcl_telemetered_driver,/omc_data/whoi/OMC/CP02PMCO/D00010/3dmgx3/*_*.3dmgx3.log,CP02PMCO-SBS01-01-MOPAK0000,telemetered,Available,
+mi.dataset.driver.WFP_ENG.STC_IMODEM.wfp_eng_stc_imodem_telemetered_driver,/omc_data/whoi/OMC/CP02PMCO/D00010/imm/mmp/E*.DAT,CP02PMCO-WFP01-00-WFPENG000,telemetered,Available,
+mi.dataset.driver.vel3d_k.wfp_stc.vel3d_k_wfp_stc_telemetered_driver,/omc_data/whoi/OMC/CP02PMCO/D00010/imm/mmp/A*.DEC,CP02PMCO-WFP01-01-VEL3DK000,telemetered,Available,
+mi.dataset.driver.dofst_k.wfp.coastal_dofst_k_wfp_telemetered_driver,/omc_data/whoi/OMC/CP02PMCO/D00010/imm/mmp/C*.DAT,CP02PMCO-WFP01-02-DOFSTK000,telemetered,Available,
+mi.dataset.driver.ctdpf_ckl.wfp.coastal_ctdpf_ckl_wfp_telemetered_driver,/omc_data/whoi/OMC/CP02PMCO/D00010/imm/mmp/C*.DAT,CP02PMCO-WFP01-03-CTDPFK000,telemetered,Available,
+mi.dataset.driver.flort_kn.stc_imodem.flort_kn__stc_imodem_telemetered_driver,/omc_data/whoi/OMC/CP02PMCO/D00010/imm/mmp/E*.DAT,CP02PMCO-WFP01-04-FLORTK000,telemetered,Available,

```

Comment:
Removed comments about not ingested due to system error. Not sure why the whole file appears that it was changed